docs/events.md

```diff
@@ -0,0 +1,88 @@
+# Player Events
+A player event is a string within the player that represents an event.
```

Comment:
The player event system uses an event target like API to register, unregister and dispatch events.